roles/client_side_tests/tasks/test_collectd.yml

```
@@ -1,13 +1,17 @@
 ---
-- name: "RHELOSP-60406"
+- name: |
+    "RHELOSP-60406"
```

There's no need for the quotation marks
```suggestion
    RHELOSP-60406
```